WalletWasabi/Blockchain/Transactions/AllTransactionStore.cs

```diff
@@ -261,6 +260,19 @@ public IEnumerable<SmartTransaction> ReleaseToMempoolFromBlock(uint256 blockHash
 
 		public IEnumerable<SmartLabel> GetLabels() => GetTransactions().Select(x => x.Label);
 
+		public async ValueTask DisposeAsync()
+		{
+			if (MempoolStore is { } mempoolStore)
+			{
+				await mempoolStore.DisposeAsync().ConfigureAwait(false);
+			}
+
+			if (ConfirmedStore is { } confirmedStore)
```

Comment:
`ConfirmedStore` cannot be `null`, can it?

Reply:
ACK